packages/imtr/src/transform.ts

```diff
@@ -79,7 +80,7 @@ export default async (argv: Props) => {
       const { slug, permits } = topic;
       const topicJsonContent = {
         permits: await Promise.all(
-          permits.map(async (permitId) => {
+          permits.map(async (permitId, count) => {
```

Comment:
Get the order of the permit id from the config here.

Reply:
Is this the count of the amount of questions or the order, than me might need to change the name.

Reply:
no it's the index of this permitId from the config.